Packages/Core/Editor/Generator/Generator.cs

```diff
@@ -70,20 +74,25 @@ public void Generate(string type, string baseWritePath, bool isEquatable, List<A
 
             foreach (var templatePath in templatePaths)
             {
-                var templateNameStartIndex = templatePath.LastIndexOf(Path.DirectorySeparatorChar) + 1;
+#if UNITY_2019_1_OR_NEWER
```

Comment:
All the changes from this line and below is not necessary, it should work with just using `templatePath`.